templates/kommander.yaml

```diff
@@ -6,6 +6,15 @@ metadata:
   namespace: kubeaddons
 spec:
   manifest: |
+    ---
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: kommander
+    spec:
+      finalizers:
+      - kubernetes
```

Comment:
What does this do?

Reply:
Ensures that cleanup occurs when the namespace is deleted